reactapp/src/Assignment2/Question12/Voting.jsx

```diff
@@ -0,0 +1,42 @@
+//Question12
+// Create a simple voting application using the useReducer hook.
+// Set up a reducer to manage the votes for different options.
+// Create buttons representing voting options.
+// Use the useReducer hook to access the votes state and dispatch actions.
+// Allow users to vote for their preferred options.
+// Display the current vote count for each option
+
+import React, { useReducer } from "react";
+
+function Voting() {
+  const reducer = (state, action) => {
+    switch (action.type) {
+      case "INCREMENT":
+        return { count: state.count + 1 };
+      default:
+        return state;
+    }
+  };
+  const initialState = { count: 0 };
+  const [stateVirat, dispatch1] = useReducer(reducer, initialState);
+  const [stateRohit, dispatch2] = useReducer(reducer, initialState);
+  const [stateDhoni, dispatch3] = useReducer(reducer, initialState);
+  return (
```

Comment:
initialState = {virat: 0, rohit: 0, dhoni: 0}
use this and update all the logic accordingly